drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -1233,6 +1117,23 @@ virtio_vdpa_dev_config(int vid)
 	}
 	priv->lm_status = VIRTIO_S_RUNNING;
 
+	virtio_vdpa_lcore_id = rte_get_next_lcore(virtio_vdpa_lcore_id, 1, 1);
+	notify_work = rte_zmalloc(NULL, sizeof(*notify_work), 0);
+	if(!notify_work) {
+		DRV_LOG(ERR, "%s vfid %d failed request memory", vdev->device->name, priv->vf_id);
+		rte_errno = rte_errno ? rte_errno : ENOMEM;
+		return -rte_errno;
+	}
+
+	notify_work->priv = priv;
+	notify_work->vq_idx = RTE_VHOST_QUEUE_ALL;
+	DRV_LOG(INFO, "%s vfid %d launch notifier work lcore:%d", vdev->device->name, priv->vf_id, virtio_vdpa_lcore_id);
+	ret = rte_eal_remote_launch(virtio_vdpa_dev_notifier_work, notify_work, virtio_vdpa_lcore_id);
+	if (ret) {
+		DRV_LOG(ERR, "%s vfid %d failed launch notifier work ret:%d lcore:%d", vdev->device->name, priv->vf_id, ret, virtio_vdpa_lcore_id);
```

Comment:
line 1130 and 1133 seems too long. maybe using two lines?

Reply:
done